### ISSUE TITLE
RabbitMQ max queue length is 255 not 127

### ISSUE THREAD

**Issue by dzen** [gmr/pamqp#24]
PAMQP limits the queue name to 127, but rabbit one is 255
https://www.rabbitmq.com/queues.html

The checks is wrong imho

**Comment by gmr** [comment]
Interesting, then it's a problem in the spec they provide, since that's a code-gen value. I do have an override document and can fix that there.

**Comment by gmr** [comment]
From the spec:

```xml
  <domain name = "queue-name" type = "shortstr" label = "queue name">
    <doc>
      The queue name identifies the queue within the vhost.  In methods where the queue
      name may be blank, and that has no specific significance, this refers to the
      'current' queue for the channel, meaning the last queue that the client declared
      on the channel.  If the client did not declare a queue, and the method needs a
      queue name, this will result in a 502 (syntax error) channel exception.
    </doc>
    <assert check = "length" value = "127" />
    <assert check = "regexp" value = "^[a-zA-Z0-9-_.:]*$" />
  </domain>
```

**Comment by gmr** [comment]
Addressed in 3.0.0a6
